batch-indexing/src/main/java/data/IndexingApplication.java

```diff
@@ -0,0 +1,13 @@
+package data;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class IndexingApplication {
+
+    public static void main(String[] args) {
```

Comment:
I think this needs to be applied similarly to batch-transformation.
If I remember correct if we leave it like that it creates some non-exiting pod, something like that.

Reply:
changed